### PR TITLE
Fix Google BigQuery sigpipe error on Exploratory Desktop

### PR DESCRIPTION
**Pull request by hidekoji**
### Description

Fix BigQuery: Importing through cloud storage throws SIGPIPE error 

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/system.R [hunk 1]
```diff
@@ -913,9 +913,10 @@ executeGoogleBigQuery <- function(project, sqlquery, destination_table, page_siz
     # check if the query contains special key word for standardSQL
     # If we do not pass the useLegaySql argument, bigrquery set TRUE for it, so we need to expliclity set it to make standard SQL work.
     isStandardSQL <- stringr::str_detect(sqlquery, "#standardSQL")
+    browser()
```
Comment (hideaki) [inline R/system.R]: Please remove browser()

File: R/system.R [hunk 2]
```diff
@@ -772,7 +772,7 @@ submitGoogleBigQueryJob <- function(project, sqlquery, destination_table, write_
   # If we do not pass the useLegaySql argument, bigrquery set TRUE for it, so we need to expliclity set it to make standard SQL work.
   isStandardSQL <- stringr::str_detect(sqlquery, "#standardSQL")
   # set envir = parent.frame() to get variables from users environment, not papckage environment
-  job <- bigrquery::insert_query_job(GetoptLong::qq(sqlquery, envir = parent.frame()), project, destination_table = destination_table, write_disposition = write_disposition, useLegacySql = isStandardSQL == FALSE)
+  job <- bigrquery::insert_query_job(GetoptLong::qq(sqlquery, envir = parent.frame()), project, destination_table = destination_table, write_disposition = write_disposition, use_legacy_sql = isStandardSQL == FALSE)
```
Comment (hideaki) [inline R/system.R]: What is this change on the argument name about?

File: R/system.R [hunk 3]
```diff
@@ -829,7 +829,7 @@ downloadDataFromGoogleCloudStorage <- function(bucket, folder, download_dir, tok
     }
   });
   files <- list.files(path=download_dir, pattern = ".gz");
-  df <- lapply(files, function(file){readr::read_csv(stringr::str_c(download_dir, "/", file))}) %>% dplyr::bind_rows()
+  df <- lapply(files, function(file){readr::read_csv(stringr::str_c(download_dir, "/", file), progress = FALSE)}) %>% dplyr::bind_rows()
```
Comment (hideaki) [inline R/system.R]: Comment on why progress=FALSE is necessary would help.

File: R/system.R (hunk 2 above)
Comment (hidekoji) [reply R/system.R]: fixed typo on argument name.

**Pull request merged**